tests/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 include_directories ("${PROJECT_SOURCE_DIR}/include")
+file(COPY testdata/test.fifu DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Do we want to put this in the binary directory? Note that CMake allows you to provide arguments to tests.

Reply:
The embeddings end up in the `test` directory, do you prefer having a dedicated `testdata` directory?

Reply:
Maybe test/data?